### PR TITLE
Disable travis in HCO. Moved to github action

### PR DESCRIPTION
**Pull request by nunnatsa**
Signed-off-by: Nahshon Unna-Tsameret <nunnatsa@redhat.com>

### CODE REVIEW

File: github/ci/prow/files/config.yaml [hunk 1]
```diff
@@ -275,7 +275,7 @@ branch-protection:
               protect: true
               required_status_checks:
                 contexts:
-                  - continuous-integration/travis-ci/pr
+                  # - continuous-integration/travis-ci/pr
```
Comment (dhiller) [inline github/ci/prow/files/config.yaml:278]: Maybe rather just delete the line, git will remember it just in case :)

**Review by dhiller — COMMENTED** [review]
One comment inside.

File: github/ci/prow/files/config.yaml (hunk 1 above)
Comment (nunnatsa) [reply github/ci/prow/files/config.yaml]: Done

**Review by dhiller — APPROVED** [review]
/lgtm
/approve

**Pull request merged**